src/firmware/setup_udev_rules.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+######################################################################
+# This script will add udev rules for the sake of persistent device  #
+# naming                                                             #
+######################################################################
+
```

Comment:
Can delete, has been moved to `setup_scripts`.

Reply:
done